arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -78,35 +78,127 @@ std::string const statsSamplesQuery = "/*statsSample*/ FOR s IN @@collection FIL
+    BytesReceivedScale, "Bytes received for a request");
+DECLARE_HISTOGRAM(arangodb_client_connection_statistics_bytes_sent,
+    BytesSentScale, "Bytes sent for a request");
+DECLARE_COUNTER(arangodb_process_statistics_minor_page_faults_total,
+    "The number of minor faults the process has made which have not required loading a memory page from disk. This figure is not reported on Windows");
+DECLARE_COUNTER(arangodb_process_statistics_major_page_faults_total,
+    "On Windows, this figure contains the total number of page faults. On other system, this figure contains the number of major faults the process has made which have required loading a memory page from disk");
+DECLARE_GAUGE(arangodb_process_statistics_user_time,
+    double, "Amount of time that this process has been scheduled in user mode, measured in seconds");
+DECLARE_GAUGE(arangodb_process_statistics_system_time,
+    double, "Amount of time that this process has been scheduled in kernel mode, measured in seconds");
+DECLARE_GAUGE(arangodb_process_statistics_number_of_threads,
+    double, "Number of threads in the arangod process");
+DECLARE_GAUGE(arangodb_process_statistics_resident_set_size,
+    double, "The total size of the number of pages the process has in real memory. This is just the pages which count toward text, data, or stack space. This does not include pages which have not been demand-loaded in, or which are swapped out. The resident set size is reported in bytes");
+DECLARE_GAUGE(arangodb_process_statistics_resident_set_size_percent,
+    double, "The relative size of the number of pages the process has in real memory compared to system memory. This is just the pages which count toward text, data, or stack space. This does not include pages which have not been demand-loaded in, or which are swapped out. The value is a ratio between 0.00 and 1.00");
+DECLARE_GAUGE(arangodb_process_statistics_virtual_memory_size,
+    double, "On Windows, this figure contains the total amount of memory that the memory manager has committed for the arangod process. On other systems, this figure contains The size of the virtual memory the process is using");
+DECLARE_GAUGE(arangodb_client_connection_statistics_client_connections,
+    double, "The number of client connections that are currently open");
+DECLARE_HISTOGRAM(arangodb_client_connection_statistics_connection_time,
+    ConnectionTimeScale, "Total connection time of a client");
+DECLARE_HISTOGRAM(arangodb_client_connection_statistics_total_time,
+    ConnectionTimeScale, "Total time needed to answer a request");
+DECLARE_HISTOGRAM(arangodb_client_connection_statistics_request_time,
+    RequestTimeScale, "Request time needed to answer a request");
+DECLARE_HISTOGRAM(arangodb_client_connection_statistics_queue_time,
+    RequestTimeScale, "Request time needed to answer a request");
+DECLARE_HISTOGRAM(arangodb_client_connection_statistics_io_time,
+    RequestTimeScale, "Request time needed to answer a request");
```

Comment:
Description copy paste error? "Request time needed to answer a request"

Reply:
Yes, I took this over from the old setup. Corrected now.